src/api/menu/components/MenuComponent.vue

```diff
@@ -65,6 +65,12 @@
 import popupMenuMixin from '../mixins/popupMenuMixin';
 export default {
   mixins: [popupMenuMixin],
-  inject: ['options']
+  inject: ['options'],
+  computed: {
+    getLabel() {
```

Comment:
same nitpick (working backwards). `get` to me implies a method call.

Reply:
This one too